src/QsCompiler/Tests.Compiler/TestCases/FunctorGeneration.qs

```diff
@@ -680,21 +680,21 @@ namespace Microsoft.Quantum.Testing.FunctorGeneration {
 
     operation InvalidAutoInversion9(q : Qubit) : Unit {
         body (...) {
-            for (i in 1..10) { set _ = 1; }
+            for i in 1..10 { set _ = 1; }
         }
         adjoint auto;    
     }
 
     operation InvalidAutoInversion10(q : Qubit) : Unit {
         body (...) {
-            using (c = Qubit()) { return (); }
+            using c = Qubit() { return (); }
```

Comment:
It would be good to coordinate when this PR is shipped with the PR that deprecates using and introduces non-block qubit allocation statements. I don't want to force people to update usings twice, and that also makes it clearer why we are doing this change. Could you please retarget this PR to a feature branch?